### PR TITLE
WE-339 build post rating functionality

### PR DESCRIPTION
**Pull request by logansparlin**
## What changed
- Added post rating functionality
- Added context directory and AlertsContext to handle snackbars

## How to verify
- Run locally or visit the deploy preview and submit feedback on a content page. You should be presented with a snackbar when the feedback is submitted
- I'm not 100% sure the best way to test the segment tracking so if someone wants to help me out with that that would be great

### CODE REVIEW

File: components/site/feedback.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    });
+    createAlert();
+  }
+
+  const submitNegativeFeedback = () => {
+    setLoadingNegative(true);
+    track('User Feedback', {
+      url: asPath,
+      docs: asPath.includes('momentum') ? 'momentum' : 'support',
+      helpful: 'No',
```
Comment (jonambas) [inline components/site/feedback.tsx:88]: Should these be booleans instead of strings?

**Review by jonambas — COMMENTED** [review]
this all looks good, but not sure how to verify 100%, will need to ask Kemper for Segment credentials

File: components/site/feedback.tsx (hunk 1 above)
Comment (jonambas) [reply components/site/feedback.tsx:88]: I verified the POST is being called, but can't verify on the Mixpanel end

**Review by jonambas — APPROVED** [review]
This looks good, should we add a timeout so that the snackbars disappear after a few seconds?

**Review by EvanYoung-SP — APPROVED** [review]
I verified that the user feedback is being submitted to segment. LGTM!

**Pull request merged**